apertium-spa.spa.dix

```diff
@@ -54983,6 +54989,7 @@
 <e lm="bombear">         <p><l>bombé</l>     <r>bombe</r></p><par n="abandón/ar__vblex"/></e>
 <e lm="bonificar" a="gema"><i>bonifi</i><par n="ata/car__vblex"/></e>
 <e lm="bonificar eo" a="gema"><p><l>bonifí</l><r>bonifi</r></p><par n="atá/car__vblex"/></e>
+<e lm="boquear">	 	 <i>Boque</i><par n="abandón/ar__vblex"/></e>
```

Comment:
Verbs in Spanish are a bit tricky. We usually need two entries. This verb should be:
`<e lm="boquear">       <i>boque</i><par n="abandon/ar__vblex"/></e>`
`<e lm="boquear">       <p><l>boqué</l>   <r>boque</r></p><par n="abandón/ar__vblex"/></e>`
The second one is for forms with diacritics: `boquéalo`.